tests/fe/get_fe_by_name_01.cc

```diff
@@ -140,6 +140,19 @@ main()
   gen.generate_all_codim("FE_Bernstein(2)");
   gen.generate_all_codim("FE_DGQHermite(3)");
 
+  // Simplex finite elements
+  gen.generate_all_codim("FE_SimplexP(1)");
+  gen.generate_all_codim("FE_SimplexP(2)");
+  gen.generate_all_codim("FE_SimplexDGP(1)");
+  gen.generate_all_codim("FE_SimplexDGP(2)");
+  gen.generate_all_codim("FE_SimplexP_Bubbles(2)");
+
+  // Mixed grids finite elements
+  gen.generate<3, 3>("FE_PyramidP(1)");
+  gen.generate<3, 3>("FE_PyramidDGP(1)");
+  gen.generate<3, 3>("FE_WedgeP(1)");
+  gen.generate<3, 3>("FE_WedgeDGP(1)");
```

Comment:
You can test quadratic wedges also.

Reply:
The mechanism that provides the degree is the same as for all other FE, and it's tested elsewhere. Do you want me to add this here nonetheless?